src/liboslexec/llvmutil_test.cpp

```diff
@@ -220,7 +220,12 @@ test_isa_features()
     // Make sure it matches what OIIO's cpuid queries reveal
     OIIO_CHECK_EQUAL(ll.supports_avx(), OIIO::cpu_has_avx());
     OIIO_CHECK_EQUAL(ll.supports_avx2(), OIIO::cpu_has_avx2());
+#if OSL_LLVM_VERSION >= 80
     OIIO_CHECK_EQUAL(ll.supports_avx512f(), OIIO::cpu_has_avx512f());
+    // N.B. Skip this check because for LLVM < 8, we intentionally disable
+    // AVX512 support in order to avoid a performance problem, so we
+    // expect this to potentially not match.
+#endif
 }
 
```

Comment:
Suggest leaving supports alone and accurate for all LLVM versions and instead during backend's selection of a target isa limit it to SSE4.2 when OSL_LLVM_VERSION < 80.